core/services/wifi/WifiManager.py

```diff
@@ -141,18 +141,24 @@ async def set_wifi_password(self, credentials: WifiCredentials) -> Any:
             network_number = data.decode("utf-8")
             await self.wpa.send_command_set_network(str(network_number), "ssid", '"{}"'.format(credentials.ssid))
             await self.wpa.send_command_set_network(str(network_number), "psk", '"{}"'.format(credentials.password))
-            await self.wpa.send_command_enable_network(network_number)
-            await self.wpa.send_command_reconnect()
-
-            answer = await self.wpa.send_command_save_config()
-            answer = answer.strip()
-            if answer == b"FAIL":
-                raise RuntimeError(str(answer))
-
+            await self.wpa.send_command_save_config()
```

Comment:
why are you ignoring save config return ? What happens if the operation fail ?

Reply:
I removed the fail check for the individual operations and added it as default on send_command so every operation is checked. Anyway, I should have removed the check on the save_config together with this other commit. Will change.

Reply:
Fixed.